Fixed an issue when fetch-indicators command had a wrong default value for "indicator type" (#12359)
demisto/content: Packs/FeedIntel471/ReleaseNotes/1_1_3.md

#### Integrations
##### Intel471 Actors Feed

> - Fixed an issue when ***fetch-indicators*** command had a wrong default value for "indicator type".

Reviewer: ```suggestion
- Fixed an issue where the ***fetch-indicators*** command had an incorrect default value for "indicator type".
```
